modules/ROOT/pages/configure-api-task.adoc

```diff
@@ -52,16 +52,20 @@ image::create-endpoint-api-with-proxy.png[Create API endpoint with proxy]
 +
 Otherwise, you can skip this step.
 .. In the *TlS Context for outbound traffic* field, click *Add TLS Context* and specify the TLS context to secure outbound traffic.
-.. In the Path field, specify the path.
+.. In *Path*, specify the path.
 .. Click *Advanced options* to specify additional settings.
-... From the *Proxy version* list, choose the proxy version that you are currently using.
-... In the *Validations* field, check the box if you want to validate inbound requests against a specification. You can optionally specify to use query parameters for strict validations.
-... In the *Scheme* field, specify whether you want to use an HTTP or an HTTPs schema.
-... In the *Port* field, specify the port number to use for the requests.
+... From *Proxy version*, choose the proxy version that you are currently using.
+... * If you want to simulate your API specification, select *Enable Console* and specify a different path in *Console Path*, for example, "/spec/*".  
```

Comment:
I don't think `simulate` is the correct word here. Also, the `specify a different path in *Console Path*` is not necessary for that "simulation" because there is a default path.

Reply:
What do you think @tutecano1995 ?